x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/SourceGenerator.java

```diff
@@ -187,8 +187,5 @@ private static void optimize(QueryContainer query, SearchSourceBuilder builder)
 
     private static void disableSource(SearchSourceBuilder builder) {
         builder.fetchSource(FetchSourceContext.DO_NOT_FETCH_SOURCE);
```

Comment:
Is there any downside to disabling stored fields?

Reply:
I hope not @costin . Tests passed. I don't see why we would want to return stored fields... can't think of a reason why we had it there in the first place. In theory, we shouldn't need any kind of way of returning values from Elasticsearch except "fields".

Reply:
The why not disable explicitly stored fields:
```
builder.storedFields(NO_STORED_FIELD);
```

Reply:
Ok. Will try that.

Reply:
@costin I forgot about this part. "fields" API does not allow disabling stored fields.